plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/procedure/TestIcebergMigrateProcedure.java

```diff
@@ -168,6 +168,38 @@ public void testMigrateTableWithComplexType(IcebergFileFormat fileFormat)
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testMigrateTimestampHiveTableInComplexType()
+    {
+        String inputValue = "2021-01-01 10:11:12.123"; // Tests are run with America/Bahia_Banderas timezone
+        String expectedValue = "2021-01-01 16:11:12.123000 UTC";
+
+        String tableName = "test_migrate_timestampComplexType_" + randomNameSuffix();
```

Comment:
Please don't mix up snake case and camel case.
```suggestion
        String tableName = "test_migrate_timestamp_complex_type_" + randomNameSuffix();
```